Added Stacked Bar and Column Charts

dcrousso opened the pull request:
I added stacked bar and column charts.  I also added a demo page and included screenshots of the new stacked charts in the readme.

dcrousso committed:
Added Stacked Bar Charts

dcrousso committed:
Added Stacked Column Charts

dcrousso committed:
Updated Readme

dcrousso committed:
Added demo page

dcrousso committed:
Added Stacked charts to base file

dcrousso committed:
Added Stacked bar/column charts to base less file

asciimoo commented:
Awesome, thanks! Could you add the style modifications to the less files too and update the index.html only on the `gh-pages` branch?

dcrousso committed:
Removed index.html

dcrousso commented:
I already added the changes to the less files (d1b99d5) and I moved the index.html from master to gh-pages.  I actually don't know how to update a pull request, so the new commits are here:
https://github.com/dcrousso/cssplot/tree/gh-pages

asciimoo commented:
Thanks!
The pull request update is automatic
